### ISSUE TITLE
Poppins installation blocked on macOS

### ISSUE THREAD

**Issue by davelab6** [google/fonts#1827]
> I am having trouble installing Poppins on my MacBook - do you know why this is? I have installed several other fonts but this is the one I really need! Thanks!

– https://twitter.com/JonnyDHerring/status/1087701730274496512

**Comment by laerm0** [comment]
Hm, works fine for me (OS X 10.13.6). Font Book validates them with no issues. I'll contact the reporter on twitter.

<img width="763" alt="window 2019-02-11 at 13 46 46" src="https://user-images.githubusercontent.com/10780161/52585768-b5b79e00-2e03-11e9-9fbd-a06d6f23c33a.png">

**Comment by fgm** [comment]
Could it just be that you cannot download it because you can't connect to fonts.googleapis.com ? This is what I've been noticing for days too.

**Comment by mariel-reed** [comment]
![Screen Shot 2019-12-05 at 9 44 07 PM](https://user-images.githubusercontent.com/38766810/70298987-6387a500-17a8-11ea-9b67-27b091085283.png)
I'm experiencing a similar issue -- was there a fix? Trying to add Poppins in fontbook, but it's coming out jumbled.

**Comment by thlinard** [comment]
Oh, I see! I presume "jumbled" designate the Devanagari sample. The fonts have a nameID 19 (Sample Text) entry, with a sample text in Devanagari (and macOS honors the nameID in displaying a sample text in Devanagari, not in English as it's usual).

**Comment by mariel-reed** [comment]
Yes - "jumbled" is the sample. Any idea why this is happening and how to fix it? Thank you!

**Comment by thlinard** [comment]
[According to its presentation](https://fonts.google.com/specimen/Poppins), Poppins has “support for the Devanagari and Latin writing systems”.

The sample isn’t jumbled. It’s a sample in Devanagari. Please, take a moment to learn some basic facts about this script: https://en.wikipedia.org/wiki/Devanagari

 There nothing wrong for a typeface like Poppins to have its sample in Devanagari. You can use it for Latin script nonetheless.

**Comment by peterbroomfield** [comment]
I've been having a similar problem with Poppins. It installs fine, but the Font Book preview is in Devanagari and then when I attempt to use the font, the glyphs appear highlighted with that _"you're missing these characters"_
<img width="1948" alt="Screen Shot 2020-10-11 at 10 09 42 PM" src="https://user-images.githubusercontent.com/53009735/95698340-a360b980-0c0f-11eb-9f6e-67cc96c66fe6.png">
 background.

**Comment by thlinard** [comment]
> I've been having a similar problem with Poppins. It installs fine, but the Font Book preview is in Devanagari and then when I attempt to use the font, the glyphs appear highlighted with that _"you're missing these characters"_ background.

Font Book preview in Devanagari isn't a problem. But in your screenshot, you seek a missing "Poppins⬧" (probably an old version, with a wrong character in the name), and the installed typeface is "Poppins".

**Comment by darchitectlp** [comment]
I disagree. Devanagari preview in Font Book is a problem. I spent an hour thinking I'd installed a corrupted font and kept trying to track down a working .ttf file... only to realize the font DOES actually work. This is a UX problem. When you see the font on googlefonts and all that is being displayed as Latin characters (with a tiny bit of Devanagari at the bottom), you expect Font Book's install window to show you the Latin characters, like it does for every other font. But it just shows a blank window. That alone made me think it was corrupt. And once it's installed, it only previews in Devanagari, which is a problem since even with my font language preferences set specifically to English, it shows the glyphs. This should be fixed. Lovely font, though!

**Comment by thlinard** [comment]
@davelab6 If you want to avoid reports like these, I think `nameID 19` should be corrected. Presently, the `langID` associated is `0x409` (i.e. US English), it should be `0x0439` (Hindi), and perhaps another `namerecord` with `nameID 19` and  `langID 0x409` should be created, with a sample in English. Or completely remove `nameID 19`, although that would be a shame ...

**Comment by Chad-Baker** [comment]
In Font Book, you can choose View > Custom (instead of Sample or Repertoire) and you should be presented with sample text in English, if that's your language. You can also substitute your own sample text or pangrams there by typing them in. I too was confused by the Devanagari display, which makes it difficult to judge the suitability of the font for Latin-based applications.
![Screen Shot 2021-11-04 at 5 52 35 PM](https://user-images.githubusercontent.com/32308931/140440286-324dafd1-8830-4b1a-8a91-f1a3cc9465fd.png)

**Comment by phunkyD** [comment]
i'm having a similar issue with POPPINS font. The issue is no matter how many times I restart and reload the ofbtp options the regular style won’t register and shows up as Hindi. i'm on mac os big sur 11.7.6.  
![Screen Shot 2023-05-02 at 8 29 07 AM](https://user-images.githubusercontent.com/91766255/235725807-ee8ee07b-17af-4288-a4ad-72045a5b93f6.jpeg)

**Comment by kenmcd** [comment]
> i'm having a similar issue with POPPINS font. The issue is no matter how many times I restart and reload the ofbtp options the regular style won’t register and shows up as Hindi. i'm on mac os big sur 11.7.6.

Indian Type Foundry, the original developers, have a Latin-only version available in their repository called Poppins Latin.
They removed the Devanagari script characters/glyphs, and the Devanagari sample text.
Download here:
https://github.com/itfoundry/Poppins/tree/master/products 
The zips are named
- `PoppinsLatin-5.001-Latin-OTF.zip` and 
- `PoppinsLatin-5.001-Latin-TTF.zip`.

So if you only need the Latin script, those fonts may work for you.

**Comment by phunkyD** [comment]
well, after some testing i decided to upgrade to Ventura and that seemed to solve my issue. POPPIN normal is now displaying properly. thank you for the options @kenmcd. before the upgrade, i created a new user on big sur 11.7.6 and the issue persisted. spoke with a user who was on Ventura they were able to view POPPIN normal properly.

**Comment by phunkyD** [comment]
sorry, POPPIN regular that is.

**Comment by barnahulumulia** [comment]
SOLVED guys, my issue is the same above. I'm using font book.app (mac). Open it, look at the "All Fonts" find your troubled font mine is POPPINS -> and the display is poppin (turn off). Turn it on by : select the font (poppins) -> click edit (top left corner bar) -> click 'Enable'

You also can remove duplicated fonts on your list, to minimizing the error i believe.
